src/mixins/date-mixin.ts

```diff
@@ -1,360 +1,206 @@
 import Vue from 'vue'
 import { Component } from 'vue-property-decorator'
 import { Getter } from 'vuex-class'
-import { isDate } from 'lodash'
 import { ApiDateTimeUtc, IsoDatePacific } from '@bcrs-shared-components/interfaces'
-
-const MS_IN_A_DAY = (1000 * 60 * 60 * 24)
+import DateUtilities from '@/resources/date-utilities'
 
 /** Mixin that provides some useful date utilities. */
 @Component({})
 export default class DateMixin extends Vue {
   @Getter getCurrentJsDate!: Date
 
   /**
+   * DEPRECATED - call resources/date-utilities instead
```

Comment:
Please create a Tech Debt ticket so we can go in and update the code that uses these deprecated methods.

Also, FYI, this Date Mixin is also used in Create UI and Edit UI, and there's even a version in BCRS Shared Components. They're all a bit different, though I'd like to have a single version (ideally, shared) that all can simply import... or whatever that looks like using Date Utilities instead.

Reply:
The reason we have 3 repos is because they have different architectures, and to be able to work on them and deploy independently.

I am open to new ideas! How do you envision keeping the UIs separate in one repo?

Reply:
There's definitely some complexity to this change, but I'd start by having a routes for edit-ui, create-ui and filings-ui.  Then simplify `App.vue` to call only `<router-view></router-view>`.  The "kick off logic" that currently resides in `App.vue` should be moved to actions and the component that the router file initially calls should trigger the appropriate action.

Reply:
So you're thinking of combining our 3 UIs into 1 (huge) UI? I see more negatives than positives...
- positive: easier to share some common code
- positive: routing is much faster than redirecting
- negative: huge app to load (unless we can load parts on-demand)
- negative: increased risk/incidence of code conflicts due to several developers working on it
- negative: can't release 3 parts independently

I've seen chatter in StackOverflow about multiple UIs in the same repo (specifically so they could share code).

And there's always the shared repo that we could move more code to.

Reply:
Thanks @severinbeauvais.  Perhaps I'm being too optimistic, but I see more positives than negatives.  By moving the kick-off-logic to actions that are triggered by specific components we can keep the amount of data that's loaded at startup to no more than present.  By separating concerns and being careful that tickets don't overlap we can avoid merge conflicts.  Let's keep the discussion going.  I SO appreciate you being open to discussing alternative ideas.

Reply:
As requested, ticket created:  bcgov/entity#15234

Reply:
Yes, it would be nice to have a code architecture where everything isn't happening at once :)